packages/components/src/TabBar/TabBar.scss

```
@@ -10,13 +10,62 @@
 	background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M8 13.6l-8-8L2.6 3 8 8.4 13.4 3 16 5.6z" fill="#{hex-to-url($color)}"/></svg>');
 }
 
+$underline-height: 2px;
```

rem?

---

And I would suggest using a better const name like
```suggestion
$tc-tabbar-active-border-size: .2rem !default;
```